src/app/__init__.py

```
@@ -1,7 +1,9 @@
-"""Flask application factory."""
+"""Flask application factory for the SVG Translate web interface."""
 
 from __future__ import annotations
 
+import threading
```

Flake8 (7.3.0)

[error] 'threading' imported but unused

(F401)